controller/backup_controller.go

```diff
@@ -811,3 +811,9 @@ func (bc *BackupController) syncBackupStatusWithSnapshotCreationTimeAndVolumeSiz
 
 	backup.Status.SnapshotCreatedAt = snap.Created
 }
+
+func (bc *BackupController) backupNotInFinalState(backup *longhorn.Backup) bool {
+	return backup.Status.State != longhorn.BackupStateCompleted &&
+		backup.Status.State != longhorn.BackupStateError &&
+		backup.Status.State != longhorn.BackupStateUnknown
+}
```

Comment:
nit: It would be nice to add a comment for the final states and non-final states in the const definition.

```
const (
        // non-final states
	BackupStateNew        = BackupState("")
	BackupStatePending    = BackupState("Pending")
	BackupStateInProgress = BackupState("InProgress")
        // final states 
	BackupStateCompleted  = BackupState("Completed")
	BackupStateError      = BackupState("Error")
	BackupStateUnknown    = BackupState("Unknown")
)
```

Reply:
added, thanks!